src/emcpy/utils/dateutils.py

```diff
@@ -0,0 +1,226 @@
+# coding: utf-8 -*-
+
+"""
+utilities for working with dates using datetime module (Python 2.3 or later)
+original author: Jeff Whitaker <jeffrey.s.whitaker@noaa.gov>
+"""
+
+__all__ = [
```

Comment:
in general a lot of these things seem like they would be better suited creating datetime objects...